src/liballoc/alloc.rs

```diff
@@ -56,6 +56,21 @@ pub struct Global;
 /// # Safety
 ///
 /// See [`GlobalAlloc::alloc`].
+///
+/// # Examples
+///
+/// ```
+/// use std::alloc::{alloc, dealloc, Layout};
+///
+/// unsafe {
+///     let layout = Layout::from_size_align(2, 1).unwrap();
+///     let ptr = alloc(layout);
+///
+///     println!("Pointer {:p} has data: {:b}", ptr, *(ptr as *mut u16));
```

Comment:
This is reading uninitialised memory which is Undefined Behaviour. I suggest writing a value to it first and then reading that value back.